src/data/moves_info.h

```diff
@@ -16772,6 +16772,10 @@ const struct MoveInfo gMovesInfo[MOVES_COUNT_DYNAMAX] =
         .contestCategory = CONTEST_CATEGORY_COOL,
         .contestComboStarterId = 0,
         .contestComboMoves = {0},
+        .additionalEffects = ADDITIONAL_EFFECTS({
+            .moveEffect = MOVE_EFFECT_ION_DELUGE,
+            .chance = 100,
+        }),
```

Comment:
This makes Plasma Fists get boosted by Sheer Force, which as far as I know isn't how it should work.
Tested on Showdown due to lack of access to Zeraora on cartridge, but I can't find Plasma Fists in the list of moves boosted by Sheer Force.